apps/explorer/package.json

```diff
@@ -32,7 +32,7 @@
   "dependencies": {
     "@alephium/shared": "workspace:*",
     "@alephium/token-list": "^0.0.12",
-    "@alephium/web3": "0.28.1",
+    "@alephium/web3": "0.29.1",
```

Comment:
To avoid the bugs we experienced with the mobile wallet in the past, we need to keep the dependencies of our projects always in sync.

Reply:
What about writing a simple script ran with turbo, which could check this for us? 🤔

Reply:
Yeah, I had the following npm script set up in the monorepo, but it seems to not work with the internal packages:

```
"postinstall": "manypkg check"
```

source: https://turbo.build/repo/docs/handbook/troubleshooting

Reply:
The original problem is this:
- https://github.com/Thinkmill/manypkg/issues/72

Other references:
- https://github.com/Thinkmill/manypkg/issues/147
- https://github.com/Thinkmill/manypkg/issues/182

The workaround I found (and I will push in another PR) is this: https://github.com/Thinkmill/manypkg/issues/72#issuecomment-1108801027

Reply:
Pushed a fix here: https://github.com/alephium/alephium-frontend/pull/344/commits/5af039482ef6543a83fbb4317c2c918e72939947